src/cargo/ops/fix.rs

```diff
@@ -535,10 +535,7 @@ impl FixArgs {
         if let Some(edition) = &self.enabled_edition {
             cmd.arg("--edition").arg(edition);
             if self.idioms {
-                match &edition[..] {
-                    "2018" => { cmd.arg("-Wrust-2018-idioms"); }
-                    _ => {}
-                }
+                if let "2018" = &edition[..] { cmd.arg("-Wrust-2018-idioms"); }
```

Comment:
Could this just be `if edition == "2018" { ... }`?

Reply:
Indeed, updated accordingly.